apps/ios/Shared/Views/UserSettings/DeveloperView.swift

```diff
@@ -12,30 +12,52 @@ import SimpleXChat
 struct DeveloperView: View {
     @AppStorage(DEFAULT_DEVELOPER_TOOLS) private var developerTools = false
     @AppStorage(GROUP_DEFAULT_CONFIRM_DB_UPGRADES, store: groupDefaults) private var confirmDatabaseUpgrades = false
+    @AppStorage(GROUP_DEFAULT_XFTP_SEND_ENABLED, store: groupDefaults) private var xftpSendEnabled = false
     @Environment(\.colorScheme) var colorScheme
 
     var body: some View {
         VStack {
             List {
                 Section {
+                    ZStack(alignment: .leading) {
+                        Image(colorScheme == .dark ? "github_light" : "github")
+                            .resizable()
+                            .frame(width: 24, height: 24)
+                            .opacity(0.5)
+                        Text("Install [SimpleX Chat for terminal](https://github.com/simplex-chat/simplex-chat)")
+                            .padding(.leading, 36)
+                    }
                     NavigationLink {
                         TerminalView()
                     } label: {
                         settingsRow("terminal") { Text("Chat console") }
                     }
+                    settingsRow("internaldrive") {
+                        Toggle("Confirm database upgrades", isOn: $confirmDatabaseUpgrades)
+                    }
                     settingsRow("chevron.left.forwardslash.chevron.right") {
                         Toggle("Show developer options", isOn: $developerTools)
                     }
-                    settingsRow("internaldrive") {
-                        Toggle("Confirm database upgrades", isOn: $confirmDatabaseUpgrades)
+                } footer: {
+                    (developerTools ? Text("Show: ") : Text("Hide: ")) + Text("Database IDs and Trasport isolation option.")
```

Comment:
```suggestion
                    (developerTools ? Text("Show: ") : Text("Hide: ")) + Text("Database IDs and Transport isolation option.")
```

Reply:
maybe it's better for localization to concat these strings?
"Hide database IDs and Transport isolation option."
"Show database IDs and Transport isolation option."